subprojects/packagefiles/mpdecimal/meson.build

```diff
@@ -3,7 +3,7 @@ project('mpdecimal', 'c', 'cpp',
         license: 'BSD-2-Clause',
         default_options: [
           'c_std=c99',
-          'cpp_std=c++11',
+          'cpp_std=c++14',
```

Comment:
This hunk seems unrelated, or is c++14 required on ARM for some reason?

Reply:
https://github.com/mesonbuild/wrapdb/actions/runs/2607518219

Needed for MSVC.

Reply:
I am sure that the CI compiled mpdecimal with `cpp_std=c++11` before. Did something change in Meson to make it fail when this setting is used with MSVC?

Reply:
Yes.

subprojects _std settings recently started being respected. Before, they would inherit the _std settings of the host project which in this case happens to be c++17: https://github.com/mesonbuild/wrapdb/blob/master/meson.build

Reply:
That warning should be non-fatal, this is a Meson bug

Reply:
The way in which the `mpdecimal` build machinery deals with this is to set `--std=c++11` for everything except `MSVC`.  `MSVC` automatically selects the lowest supported C++ version (currently `C++14`) if no standard is given.

`meson` could just omit the standard for `MSVC` if the selected standard is `<= C++14`.